script/Arcueid/Day 9/09_00_ARC09_2_1.txt

```diff
@@ -56,52 +68,70 @@ The results were all normal. No problems.
 [sha:00eade398fb36e0172dac454a9f0b0c1253c1e7a]{
 -- Page 52, Offset 21446.
 -- 阿良句博士が“アタシったら無駄骨チャーン！”と唇をとがらせるのもやむなしだ。
-Professor Arach couldn't help but pout as she shouts, 'What am I, Worthless-chan?!'
+// Sisyphean is more like an _endless_ task than a _futile_ one, but it's not far off?
```

Comment:
Well, I appreciate the time you put into this. but your commenting self is right, sisyphean isn't quite right. Of the options below, some of which are cute and some of which hurt me, I _guess_ I prefer "frau fruitless"... However, I wonder if we shouldn't just go back to something like "waste-of-time-chan" just cop out.